src/pages/about.astro

```diff
@@ -27,7 +27,12 @@ import Layout from "../layouts/Layout.astro";
 
         <h2>Governance and Structure</h2>
 
-        <p>Open Source Pledge is brought to you by <a href="https://sentry.io/welcome/">Sentry</a> and <a href="https://github.com/opensourcepledge/osspledge.com/graphs/contributors">contributors</a>.</p>
+        <p
+          >Open Source Pledge is brought to you by
+          <a href="https://sentry.io/welcome/">Sentry</a>
+          and
+          <a href="https://github.com/opensourcepledge/opensourcepledge.com/graphs/contributors">contributors</a>.</p
+        >
```

Comment:
Didn't we get rid of this? Need a sync from `main`?

Reply:
Nope — we got rid of it everywhere _but_ the about page.